src/test/variables/variablesTest.ts

```diff
@@ -106,7 +106,11 @@ describe('variables', () => {
       const worker = await r.worker();
       outputs.push({ output: await worker.dap.once('output'), logger: worker.logger });
       outputs.push({ output: await worker.dap.once('output'), logger: worker.logger });
-      outputs.sort((a, b) => a.output.source!.name!.localeCompare(b.output.source!.name!));
+      outputs.sort((a, b) => {
```

Comment:
noticed this test had flaked on master with `cannot read property name of undefined`